pages/common/uniq.md

```diff
@@ -0,0 +1,19 @@
+# uniq
+
+> Report or omit repeated lines.
+
+- Display each line once.
+
+`uniq {{file}}`
+
+- Display occurence of each line along with that line.
+
+`uniq -c {{file}}`
+
+- Display each duplicate line once.
+
+`uniq -d {{file}}`
+
+- Display unique lines.
```

Comment:
I would rather change description to the following:
"Display only unique lines"